lib/gettext_i18n_rails_js/task.rb

```diff
@@ -103,11 +103,19 @@ def files_list
     end
 
     def output_path
-      ::Rails.root.join(
+      rails_root.join(
         GettextI18nRailsJs.config.output_path
       )
     end
 
+    def rails_root
```

Comment:
Should this be `engine_root`?

Reply:
Do you want to rename it or keep it like that?